tools/generate-mocks/generate-mocks.ts

```diff
@@ -192,7 +276,10 @@ export class ${mockClassName} {${propertiesString}${methodsString}}
 
   private getImports(components: ComponentMetaData[]): string[] {
     const importStatements = [];
-    const angularCoreImports = ['forwardRef'];
+    const angularCoreImports: string[] = [];
+    if (components.some((metaData) => ['Component', 'Directive'].includes(metaData.decorator))) {
```

Comment:
Consider assigning expression to well-named const and use that in the `if`

Reply:
Fixed 👍🏻